Fix: RabbitMQ connector example page improvements [main] (#2183)
wso2/docs-mi: en/docs/reference/connectors/rabbitmq-amqp-1.0-connector/rabbitmq-amqp-1.0-connector-example.md

        "message": "Hello World!"
    }
    ```

>     <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddAPIRequestPayload.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddAPIRequestPayload.png" title="Adding the sample API request." width="70%" alt="Adding the API request."/></a>

Reviewer: ```suggestion
    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddAPIRequestPayload.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddAPIRequestPayload.png" title="Adding the sample API request." width="50%" alt="Adding the API request."/></a>
```

#### Add RabbitMQ Connector to the Project

5. Now we will add the `publishMessage` operation of the RabbitMQ Connector to the integration flow. To do this, we need to add the RabbitMQ Connector to the integration project first.
   For that, open the **Resource View**, click on the **+** icon on the canvas to open the **Mediator Palette** and search for `RabbitMQ` in the **Mediators** section. Then, select the **RabbitMQ** connector and click on the **Download** button.

    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/SelectPublishMessageOperation.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/SelectPublishMessageOperation.png" title="Adding RabbitMQ Connector" width="80%" alt="Adding RabbitMQ Connector"/></a>

6. Click on the `publishMessage` operation to add that operation to the integration flow.

#### Create Connection

7. Create a new connection by clicking on the '+ Add new connection' button as shown below. It will open a new pop-up window.

>     <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/CreateConnection1.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/CreateConnection1.png" title="Creating a new Connection" width="70%" alt="Creating a new Connection"/></a>

Reviewer: ```suggestion
    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/CreateConnection1.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/CreateConnection1.png" title="Creating a new Connection" width="80%" alt="Creating a new Connection"/></a>
```

13. Click the **+** icon located immediately after the **Publish Message** operation to open the **Mediator Palette**. Add an [If Else Mediator]({{base_path}}/reference/mediators/filter-mediator) to verify the acknowledgment status from the RabbitMQ broker and proceed with the integration accordingly.

14. Under **Mediators**, search for `if else` and select the **If Else** mediator.

    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/SearchIfElseMediator.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/SearchIfElseMediator.png" alt="Select If Else Mediator" width="80%"></a>

    You will use an expression to define the condition for the **If Else** mediator. This condition evaluates whether the Success field in the RabbitMQ connector response is `true` to determine the appropriate flow. If `true`, the integration generates a success message and responds to the client application; otherwise, it sends an error message to the client.

> 15. In the **Add If Else Mediator** pane that appears, click on the expression editor (<img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddIfElseMediator.png" alt="inline expression editor" class="inline-icon">) icon to open the editor.

Reviewer: ```suggestion
15. In the **Add If Else Mediator** pane that appears, click on the expression editor icon (`EX`) to open the editor.

    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddIfElseMediator.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddIfElseMediator.png" alt="Add if else Mediator" width="80%"></a>
```

17. Finally, click **Add** to insert the **If Else** mediator into the integration flow.

18. Click on the **+** icon in the **Then** branch on the canvas to open the **Mediator Palette**. This branch executes when the Success field is `true`.

    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddMediatorsToThenBranch.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddMediatorsToThenBranch.png" alt="Add Mediators to then branch" width="80%"></a>

19. Under **Mediators**, select the **Log** mediator to log the successful message publishing response from the RabbitMQ broker. Configure the Log mediator to use the variable `${vars.rabbitmq_publishMessage_1.payload.messageId}` to log the message ID from the RabbitMQ response.

    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddLogMediator.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddLogMediator.png" title="Adding Log Mediator" width="70%" alt="Adding Log Mediator"/></a>

>     <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddLogMediator2.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddLogMediator2.png" title="Adding Log Mediator" width="70%" alt="Adding Log Mediator"/></a>

Reviewer: ```suggestion
    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddLogMediator2.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddLogMediator2.png" title="Adding Log Mediator" width="60%" alt="Adding Log Mediator"/></a>
```

    
20. Then select the **Payload** mediator from the **Mediator Palette** to create a custom response payload to be sent back to the client application.

    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddPayloadMediator.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddPayloadMediator.png" title="Adding Payload Mediator" width="70%" alt="Adding Payload Mediator"/></a>

>     <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddPayloadMediator2.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddPayloadMediator2.png" title="Adding Payload Mediator" width="70%" alt="Adding Payload Mediator"/></a>

Reviewer: ```suggestion
    <a href="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddPayloadMediator2.png" class="glightbox"><img src="{{base_path}}/assets/img/integrate/connectors/rabbitmq/AddPayloadMediator2.png" title="Adding Payload Mediator" width="60%" alt="Adding Payload Mediator"/></a>
```
